core/math/math_funcs.h

```diff
@@ -290,6 +290,7 @@ class Math {
 	static double random(double from, double to);
 	static float random(float from, float to);
 	static real_t random(int from, int to) { return (real_t)random((real_t)from, (real_t)to); }
+	static int random_int(int from, int to);
```

Comment:
Is there an actual need for `real_t random(int from, int to)`? Maybe we could drop it and have only `int random(int from, int to)`? Or does this cause compilation difficulties?

Reply:
Yeah, it creates a problem with compilation, and it seems the only way to solve this problem

Reply:
Ah, nvm, I think I will fix that now

Reply:
Fixed